app/src/main/java/com/simpragma/recipe/recipeapplication1/SearchItemActivity.java

```diff
@@ -0,0 +1,269 @@
+package com.simpragma.recipe.recipeapplication1;
+
+import android.app.FragmentManager;
+import android.app.FragmentTransaction;
+import android.content.Intent;
+import android.graphics.Color;
+import android.net.Uri;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.GridLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Filter;
+import android.widget.Filterable;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.DatabaseHandler;
+import com.android.volley.NetworkResponse;
+import com.android.volley.Request;
+import com.android.volley.RequestQueue;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.StringRequest;
+import com.android.volley.toolbox.Volley;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.util.ArrayList;
+
+public class SearchItemActivity extends AppCompatActivity implements Filterable,View.OnClickListener {
+
+    EditText eidiText;
+    Button button;
+    String TAG=getClass().getSimpleName();
```

Comment:
Space before and after =.

Reply:
Given space after and before assigning the value.